source/gather_info.sh

```diff
@@ -287,7 +294,18 @@ cat << EOB
         "text": {
             "copy": "https://www.alfredapp.com/ (text here to copy)",
             "largetype": "https://www.alfredapp.com/ (text here for large type)"
-        }
+        },
+    },{
+        "type": "default",
+        "uid": "acct-status",
```

Comment:
we can make this unique even though it doesn't seem to be required
```suggestion
        "uid": "group-status",
```